content/kotlin/concepts/math-methods/terms/acosh/acosh.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'acosh()'
+Description: 'Returns the hyperbolic arccosine of a numeric argument.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Kotlin'
+  - 'Math'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`acosh()`** function returns the hyperbolic arccosine of a number.
+
+## Syntax
+
+```pseudo
+acosh(x)
+```
+
+The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` datatype.
+
+It returns the hyperbolic arccosine of the argument, which represents a real number.
```

Comment:
```suggestion
The function accepts one mandatory parameter `x`, which should be a numeric value greater than or equal to `1`. The argument can be of `Double` and `Float` datatype. 

> **Note**: It returns the hyperbolic arccosine of the argument, which represents a real number.
```
We can take float datatype also. [ref](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/acosh.html)